CRM/Utils/Date.php

```diff
@@ -2447,9 +2451,11 @@ protected static function getDateElement(string $date, string $separator, int $m
    * @param string $separator
    * @param int $yearPlacement
    *
+   * @internal
+   *
    * @return string
    */
-  public static function replaceShortYear($date, string $separator, int $yearPlacement): string {
+  protected static function replaceShortYear($date, string $separator, int $yearPlacement): string {
```

Comment:
This function & the following function were only just added. We should make protected & internal unless there is a reason not to so fixing in this PR